Error when buliding sample MNIST with visual studio 2019

**Describe the bug**
Samle path:     onnxruntime/samples/c_cxx/MNIST

When  building it, got error:
$ build.bat   //this command

//////////////////////////
MNIST.cpp
MNIST.cpp(44): error C2039: "max_element": 不是 "std" 的成员
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\include\cmath(683): note: 参见 “std”的声明
MNIST.cpp(44): error C3861: “max_element”: 找不到标识符
MNIST.cpp(188): error C2039: "min_element": 不是 "std" 的成员
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\include\cmath(683): note: 参见 “std”的声明
MNIST.cpp(188): error C3861: “min_element”: 找不到标识符
MNIST.cpp(192): error C3536: “range”: 初始化之前无法使用
/////////////////////////

**System information**
- Windows 10
- ONNX Runtime installed from source
- Python version:  3.8.5
- Visual Studio version (if applicable): Microsoft Visual C++ 2019   16.7.1
- GCC/Compiler version (if compiling from source): Microsoft (R) Windows (R) Resource Compiler Version 10.0.10011.16384
- CUDA/cuDNN version: just cpu version
- GPU model and memory: just cpu version

**Expected behavior**
no errors

Maybe need translate this error message：
//////////////////////////
MNIST.cpp
MNIST.cpp(44): error C2039: "max_element": not member variable  of "std" 
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\include\cmath(683): note: reference to declaration “std” .
MNIST.cpp(44): error C3861: “max_element”:  cannt find this identifier.
MNIST.cpp(188): error C2039: "min_element": not member variables "std" 
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\include\cmath(683): note: reference to declaration “std” .
MNIST.cpp(188): error C3861: “min_element”: cannot find this identifier.
MNIST.cpp(192): error C3536: “range”:  unavailable before initialization
/////////////////////////

--- a/samples/c_cxx/MNIST/MNIST.cpp
+++ b/samples/c_cxx/MNIST/MNIST.cpp
@@ -6,6 +6,7 @@
 #include <onnxruntime_cxx_api.h>
 #include <array>
 #include <cmath>
+#include <algorithm>
 
 #pragma comment(lib, "user32.lib")
 #pragma comment(lib, "gdi32.lib")

